docs/recommendations.md

```diff
@@ -10,3 +10,14 @@ of the code in various repositories. The recommendation is to run CodeQL on ever
 pull request and on every commit to the main branch.
 
 Issue: ([#15](https://github.com/open-telemetry/sig-security/issues/15))
+
+## Binding to Network Interfaces
+Always bind to localhost rather than to 0.0.0.0 or any interface, unless there is a specific need to do otherwise. Binding to localhost reduces the attack surface of your application by making it only accessible to devices on the same machine.
+Binding to 0.0.0.0 or "all" interfaces can make your application respond on all current and future network interfaces.
+
+**Rationale:**
```

Comment:
Almost there! The "Rationale" part can be removed as well, I believe the parts prior to this cover the rationale as well.

Reply:
@jpkrohling  I have made the changes, I removed the initial file as well. Thank you!